infra/README.md

```diff
@@ -20,3 +27,56 @@ To make changes to `makemake`:
       ```
       nixos-rebuild switch --show-trace -L --flake /root/ngipkgs/#makemake
       ```
+
+## Secret management
+We use [sops-nix](https://github.com/Mic92/sops-nix) to manage secrets in
+NGIpkgs. In order to access the secrets they must be encrypted with your public
+key and you must have the matching private key available in your system.
+[Sops](https://github.com/getsops/sops) must also be installed and configured.
+
+### Viewing and editing secrets
+By default on Linux, sops expects the private keys to be in
+`$XDG_CONFIG_HOME/sops/age/keys.txt` or `$HOME/.config/sops/age/keys.txt`. You
+can generate an age key using `age-keygen` or `ssh-to-age` to convert a SSH key
+to an age compatible one. Once you've setup your key, you can run `sops -d
+<file-to-decrypt>` to view an encrypted file or `sops <file-to-edit>` to edit a
+file.
+
+### Encrypting secrets for a new keυ
```

Comment:
Note that this process is vanilla SOPS. I am not sure it pays off to explicitly describe it versus just linking to SOPS docs.

Reply:
Unless the sops docs have a summary of the same length we can link to directly, having two sentences on what to do is productive.

Reply:
[This](https://getsops.io/docs/#adding-and-removing-keys) is the upstream documentation on how to add more keys. Personally, I find there's value to document what sops usage in ngipkgs specifically looks like (hence the verbose `git diff` example).